webpack.common.config.js

```diff
@@ -21,7 +21,9 @@ module.exports = function(outputFilename) {
         'nls': '@okta/i18n/src/json',
         'okta': `${LOCAL_PACKAGES}/@okta/courage-dist/okta.js`,
         'okta-i18n-bundles': 'util/Bundles',
-        'jquery': `${LOCAL_PACKAGES}/@okta/courage-dist/jquery.js`,
+
+        // Uncomment the line below to use the version of jQuery from courage
+        // 'jquery': `${LOCAL_PACKAGES}/@okta/courage-dist/jquery.js`,
```

Comment:
I don't think we have instructions for this in the README, correct? Should we remove this code?

Reply:
I left this in only for us, in case we needed to do some quick A/B test. There is no reason an end-user should need / want to use the courage jquery